### ISSUE TITLE
Any way to find & download app without adding the collection?

### ISSUE THREAD

**Issue by HeWhoHeWho** [redsolver/skydroid#22]
As title. 

Because the whole F-droid collection has over 1k apps. I'm not gonna let them flood my main page as I already know which apps I need. Any advice?
So far I'm able to find Bromite (bromite.bdroid), the rest are not working. For instance, Snapdrop. "Snapdrop.app"/ "snapdrop.fdroid" / "snapdrop.bdroid" / "snapdrop.fdroid-app" all of these won't work. So I can't find my app without installing the Collection..

Any workaround?

**Comment by redsolver** [comment]
Snapdrop is https://to.skydroid.app/com.fmsys.snapdrop.fdroid-app

**Comment by redsolver** [comment]
If you don't want to add the full collection, you can browse the list on your computer here (https://collection.skydroid.app/fdroid-app) or guess the domain using `PACKAGE_NAME.fdroid-app`. I do recommend just adding the F-Droid collection, because you can use the search feature to search for apps easily and all apps that have updates available are always shown first, so they shouldn't flood your main page too much

**Comment by redsolver** [comment]
There is a list of all bridged F-Droid collections available here: https://codeberg.org/mondstern/F-Droid-Paketquellen/wiki/SkyDroid

**Comment by HeWhoHeWho** [comment]
Hi dev, thanks for your prompt response! I will look into them and try out each ways to see which is the best for me :)

Thanks again for the wonderful app! :)

**Comment by ambientNexus** [comment]
> There is a list of all bridged F-Droid collections available here: https://codeberg.org/mondstern/F-Droid-Paketquellen/wiki/SkyDroid

I get a 404 error whenever I try to access this link, is there something wrong with it?

**Comment by redsolver** [comment]
@newtrogicNexus yep, it was moved here: https://github.com/redsolver/skydroid/wiki/List-of-SkyDroid-Collections
